src/components/modal/README.md

```diff
@@ -1 +1,112 @@
-# Modal
\ No newline at end of file
+# Modal
+
+A modal is a small window that is displayed on top of the main application/site window. The main page is still visible, but the background is darkened to direct focus to the content of the modal window.
+
+## When to use
+
+Modals are very good for allowing users to handle small tasks related to an overarching workflow or goal. This could include:
+
+- Updating personal information as part of a larger process
+- Determining information needed in their main workflow e.g. calculating a number, looking up a code
+
+## When to use something else
+
+Please consider using the alert component if you are in need of a way to display an error message. The modal component should not be used solely for this purpose.
+
+If you need to display large amounts of information for multiple tasks consider using a dedicated page instead.
+
```

Comment:
Is this a good opportunity to include a note discouraging developers from using the modal for email newsletter sign-up overlays?